## 12.8 Confidence intervals 

### 12.8.1 Confidence intervals for the regression coefficients

Confidence intervals give us more information than the estimated regression coefficient, because they take into account the uncertainty in our estimates. Confidence intervals are constructed in such a way so that they will contain the true population parameter a specified proportion of the time, typically 95%.

A formal interpretation of the 95% confidence interval is as follows: if the analysis was repeated 100 times and 95% confidence intervals were obtained each time, then 95% of those confidence intervals would contain the true population parameter. More informally, confidence intervals are often interpreted as a range of plausible values for the true population parameter.

The definition of a  (1−𝛼) % confidence interval for the parameter  𝛽1  is given by:

𝛽1^±𝑡𝛼2,𝑛−2𝑆𝐸(𝛽1^)
 
where  𝑡𝛼2,𝑛−2  represents the  (1−𝛼/2)  centile of a  𝑡 -distribution with  (𝑛−2)  degrees of freedom.

If 0 lies in the confidence interval, we would conclude that the independent variable and outcome are not associated, because it is plausible that  𝛽1=0 . If 0 does not lie within the confidence interval, then there is evidence of an association.

Note that if  𝑛  is sufficiently large, the  𝑡 -distribution is well approximated by a normal distribution. In this case, a 95% confidence interval can be found by:

𝛽1^±1.96×𝑆𝐸(𝛽1^)
 
Example: Calculate a 95% for  𝛽1  (using the values given in the R output above):

→→95% CI:𝛽1^±1.96×𝑆𝐸(𝛽1^)95% CI:0.46656±1.96×0.0305495% CI:(0.407,0.526)
 
Since 0 does not lie within the interval, we conclude that there is evidence of an association between birthweight and length of pregancy (as indicated by the results of the hypothesis test).

Alternatively, we can obtain confidence intervals using confint in R:

In [7]:
#Confidence intervals for beta_1
confint(model1, parm=2, level = 0.95)

2.5 %     97.5 %   
Gestational.Days 0.4066435 0.5264702

*Exercise:* Edit the above code to find 95% and 90% confidence intervals for $\hat{\alpha_1}$ in Model 2. **Hint**: Use the [R documentation](https://www.rdocumentation.org/packages/stats/versions/3.6.2/topics/confint) for the command ```confint``` as a guide.

So far we have only discussed conducting inference on the estimated regression coefficients. However, it may also be of interest to determine **confidence intervals for the fitted outcomes**, or **prediction intervals**. The subsequent two sections describe and illustrate these two concepts, respectively. 

### 12.8.2 Confidence intervals for a fitted value 

Confidence intervals can be obtained for the fitted value of $Y$ given a particular value of $X$. We denote the fitted value when $X=x$ as $y_x$. The expected value of $y_x$ is equal to $\hat{Y}=\beta_0+\beta_1x$ and its variance is given by:

$$V(\hat{y}_x) = \sigma^2 (\frac{1}{n}+\frac{(x-\bar{x})^2}{SS_{xx}})$$

where $SS_{xx}=\sum_{i=1}^n(x_i-\bar{x})^2$, i.e. the sum of squares of $X$.  

Therefore, the 95% confidence interval is given by:

$$\hat{y_x} \pm t_{n-2, 0.975}\hat{\sigma} \sqrt{\frac{1}{n}+ \frac{(x-\bar{x})^2}{SS_{xx}}}$$

95% confidence intervals can be obtained for values of the independent variable that do not belong in the data. However, the width of the confidence interval increases with the distance from the mean (as can be seem from the formula and figure given below). Care must be taken when extrapolating outside the range of the observed data as this makes an un-testable assumption that linearity continues outside the observed data range. 

*Example*. The R code below calculates a 95% confidence interval for the fitted value of birthweight of a baby born after 280 gestational days. 
 

In [8]:
#Confidence interval for a fitted value 
new.data<-data.frame(Gestational.Days=280)
predict(model1, newdata=new.data, interval="confidence", level=0.95)

fit      lwr      upr     
1 119.8818 118.9215 120.8421

The 95\% confidence interval for $y_{280}$ is (118.9, 120.8). Informally, we can interpret this as: it is plausible that the true value of $y_{280}$ lies between 118.9 and 120.8.  
